app/pages/docs/css.mdx

```diff
@@ -30,26 +30,27 @@ export default function App({Component, pageProps}) {
 }
 ```
 
-These styles (`styles.css`) will apply to all pages and components in your application.
-Due to the global nature of stylesheets, and to avoid conflicts, you may **only import them inside [`_app.js`](./app-component)**.
+Deze stijlen (`styles.css`) zijn van toepassing op alle pagina's en componenten in je applicatie.
+Vanwege het globale karakter van stylesheets, en om conflicten te vermijden, mag je ze **alleen importeren binnen [`_app.js`](./app-component)**.
 
-In development, expressing stylesheets this way allows your styles to be hot reloaded as you edit them—meaning you can keep application state.
+Als je stylesheets op deze manier gebruikt, kunnen ze tijdens het ontwikkelen opnieuw geladen worden zodat je de status van de applicatie kan behouden.
 
-In production, all CSS files will be automatically concatenated into a single minified `.css` file.
+In productie zullen alle CSS-bestanden automatisch worden samengevoegd tot één verkleind `.css` bestand.
```

Comment:
Houden we hier **minified** of toch **verkleind**? Ik ben dat laatste nog niet veel tegengekomen.